selfdrive/car/card.py

```diff
@@ -109,6 +109,11 @@ def __init__(self, CI=None, RI=None) -> None:
 
       self.CI = get_car(*self.can_callbacks, obd_callback(self.params), experimental_long_allowed, num_pandas, cached_params, fixed_fingerprint)
       sunnypilot_interfaces.setup_car_interface_sp(self.CI.CP, self.CI.CP_SP, self.params)
+      sunnypilot_interfaces.set_hyundai_long_tune_flag(self.CI.CP_SP, self.params)
+
+      if self.CI.CP.brand == 'hyundai':
+        self.CI.apply_longitudinal_tuning()
+
```

Comment:
I wonder if this could be inside `setup_car_interface_sp` instead, i see we have some make specific logic there, but there's no make specific logic in card